content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -152,6 +152,30 @@ host's file owner/group.
 
 [CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
 
+## Integration into Pod Security Standards
```

Comment:
```suggestion
## Integration with Pod security admission {#pod-security-admission}
```

If you change this, also change the incoming hyperlink.

Reply:
Changed only into --> with, until the PSS vs PSA in the other thread is resolved.